packages/mongo/oplog_tailing.js

```diff
@@ -176,6 +202,18 @@ _.extend(OplogHandle.prototype, {
     var f = new Future;
     self._catchingUpFutures.splice(insertAfter, 0, {ts: ts, future: f});
     f.wait();
+
+    complete();
+
+    function complete() {
```

Comment:
might be worth pulling this function to the top near the declaration of `callback` for readability